packages/ui/src/ui/components/Icon/importGravityIcons.ts

```diff
@@ -15,4 +15,5 @@ export const iconNames = {
     ['layout-side-content']: LayoutSideContent,
     ['sql']: AbbrSql,
     ['crowndiamond']: CrownDiamond,
+    ['python']: LogoPython,
```

Comment:
It is never used directrly. Do we really need it?

Reply:
You are right, it's not neccesery, I will roll it out.